externs/shaka/player.js

```diff
@@ -129,6 +131,11 @@ shakaExtern.Stats;
  *   (only for video tracks) The width of the track in pixels.
  * @property {?number} height
  *   (only for video tracks) The height of the track in pixels.
+ * @property {?number} frameRate
+ *   (only for video tracks) The video framerate provided in the manifest.
+ * @property {?string} codecs
+ *   (only for audio and video tracks) The audio/video codecs string provided
```

Comment:
Text streams can also have a codecs string, although it is not always required.  I suggest dropping the parenthetical statement and adding the caveat "if present" after "in the manifest".